lib/ExpensiMark.js

```diff
@@ -169,6 +169,13 @@ export default class ExpensiMark {
                 },
             },
 
+            {
```

Comment:
Can we add a comment here explaining the regex the same way the other mention regex have?